framework/src/testing/create_contexts.ts

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+import { Block, GenesisBlock, Transaction } from '@liskhq/lisk-chain';
+import {
+	AfterBlockApplyContext,
+	AfterGenesisBlockApplyContext,
+	ApplyAssetContext,
+	BeforeBlockApplyContext,
+	Consensus,
+	ReducerHandler,
+	StateStore,
+	TransactionApplyContext,
+} from '../types';
+import { createGenesisBlock } from './create_genesis_block';
+import { ModuleClass } from './types';
+import { StateStoreMock } from './mocks/state_store_mock';
+import { reducerHandlerMock } from './mocks/reducer_handler_mock';
+import { consensusMock } from './mocks/consensus_mock';
+
+export const createAfterGenesisBlockApplyContext = <T = unknown>(params: {
+	modules?: ModuleClass[];
+	genesisBlock?: GenesisBlock<T>;
+	reducerHandler?: ReducerHandler;
+	stateStore?: StateStore;
+}): AfterGenesisBlockApplyContext<T> => {
+	const modules = params.modules ?? [];
+	const genesisBlock = params.genesisBlock ?? createGenesisBlock<T>({ modules, accounts: [] });
```

Comment:
will it work if i pass modules and then pass empty accounts? we don't need accounts associated with the modules props?

Reply:
Yes it should work in terms of genesis block creation. As we don't enforce number of accounts for the genesis block.